config/translate.go

```diff
@@ -82,18 +82,20 @@ func TranslateFromV1(old v1.Config) (types.Config, error) {
 		config.Storage.Filesystems = append(config.Storage.Filesystems, filesystem)
 
 		for _, oldFile := range oldFilesystem.Files {
-			file := types.File{
-				Filesystem: filesystem.Name,
-				Path:       types.Path(oldFile.Path),
+			file := types.RegFile{
```

Comment:
I feel like the abbreviation of `RegularFile` is somewhat spurious, when the `Directory` type hasn't been abbreviated, and `RegularFile` is only two characters longer.  A casual inexperienced reader may interpret `RegFile` as register file or something unexpected, just use the expanded form, it's more clear.